Assets/Scripts/New Features/PlayerJournal.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerJournal : MonoBehaviour
+{
+
+    private const string progressFormat = "{0}/{1}";
+
+    [SerializeField]
+    private GameObject playerJournal = null;
+    // Start is called before the first frame update
+
+    public List<Quest> PlayerQuests = null;
+
+    [SerializeField]
+    private GameObject panel = null;
+
+    [SerializeField]
+    private GameObject questItemPrefab = null;
+
+    #region Singleton
+    public static PlayerJournal instance;
+    private void Awake()
+    {
+        if (instance != null)
+        {
+            Debug.LogWarning("More than one instance");
+            return;
+        }
```

Comment:
-Debug logs should only be present in Unity Editor, see previous comment
-Also, if there is more than one instance, don't we want to destroy the new one if this is a singleton?